frontend/storage/reducers/index.ts

```diff
@@ -1,8 +1,10 @@
 import { AppStore, AppActions } from '../common';
 import { breadcrumbReducer } from './breadcrumbReducer';
 import { gameVisibilityReducer } from './gameVisibilityReducer';
+import { downloadReducer } from './downloadReducer';
```

Comment:
Just so you know, I don't like the name `downloadReducer` but I don't know enough about what it'll do to offer a good alternative